docs/component-guides/kserve.md

```diff
@@ -1,30 +1,34 @@
-1. Verify the current directory is the root of the repository by running the `pwd` command. The output should be `<path/to/kubeflow-manifests>` directory
+This guide assumes that you have:
 
+1. The [AWS Load Balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/) configured with one of the following deployment options:
+    - A Cognito-integrated deployment that is configured with the [AWS Load Balancer controller by default](../deployment/cognito/README.md#30-configure-ingress).
+    - A deployment that is not integrated with Cognito (for example, the [Vanilla deployment](/docs/deployment/install/vanilla/guide/), which uses Dex as an auth provider), but have followed the [Exposing Kubeflow over Load Balancer guide](../deployment/add-ons/load-balancer/README.md).
+2. A subdomain for hosting Kubeflow. For this guide, we will use the domain `platform.example.com`.
+1. An existing [profile namespace](https://www.kubeflow.org/docs/components/multi-tenancy/getting-started/#manual-profile-creation) for a user in Kubeflow. For this guide, we will use the example profile namespace `staging`.
+1. Verified that your current directory is the root of the repository by running the `pwd` command. The output should be `<path/to/kubeflow-manifests>` directory.
 
-### Configure Knative Serving's domain
 
-KFServing uses Knative Serving for various things like serverless deployment, autoscaling, setting up network routing resources etc. For purpose of this guide, we will focus on the routes. The fully qualified domain name(FQDN) for a route in Knative Serving by default is `{route}.{namespace}.{default-domain}`. Knative Serving routes use `example.com` as the default domain ([Reference](https://knative.dev/docs/serving/using-a-custom-domain/#default-domain-name-settings)). For example, if you create an `InferenceService` resource called `sklearn-iris` in `staging` namespace. Its URL would be `http://sklearn-iris.staging.example.com`. This is also the default configuration in Knative Serving component deployed as part of your Kubeflow deployment. You will need to configure the default domain as per your deployment. Let's see how:
+## Configure a domain
 
-As you understand from the [background](#background) section, we need to use a custom domain to configure HTTPS with load balancer. We recommend using HTTPS to to enable traffic encryption between the clients and your load balancer. So, if you are using `platform.example.com` domain to host Kubeflow, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes.
+Use [Knative Serving](https://knative.dev/docs/serving/) to set up network routing resources by following the [Changing the default domain](https://knative.dev/docs/serving/using-a-custom-domain/#procedure) procedure. 
```

Comment:
I like this because user knows in the first line there is an action item but this might be too early to redirect the user.

Shall we keep this line to say we will be configuring Knative servings default domain so user knows there is a n action item and

put the instruction about following the Changing the default domain section before the `Edit the ConfigMap `line